assets/sass/01-settings/global.scss

```diff
@@ -224,11 +224,11 @@ $baseline-unit: 10px;
 	--quote--border-width: 1px;
 	--quote--font-family: var(--global--font-secondary);
 	--quote--font-size: var(--global--font-size-md);
-	--quote--font-size-large: var(--global--font-size-lg);
+	--quote--font-size-large: calc(.5 * var(--global--font-size-xxl));
```

Comment:
You're basing this on the `--global--font-size-xxl` which actually bumps up to 6rem on larger screens, and I don't _think_ we want that on large quotes, it seems too large.

![Screen Shot 2020-10-15 at 11 17 38 AM](https://user-images.githubusercontent.com/541093/96150081-0ab08f00-0ed8-11eb-86f2-9a66f901f8ff.png)

Maybe this?

```suggestion
	--quote--font-size-large: calc(1.6 * var(--global--font-size-base));
```